src/Compilation.zig

```diff
@@ -1889,13 +1889,13 @@ pub fn totalErrorCount(self: *Compilation) usize {
         // the previous parse success, including compile errors, but we cannot
         // emit them until the file succeeds parsing.
         for (module.failed_decls.keys()) |key| {
-            if (key.namespace.file_scope.okToReportErrors()) {
+            if (key.getFileScope().okToReportErrors()) {
```

Comment:
Since I was updating all of the references to decl.namespace (see the comment on that declaration for why), I also figured I'd update the ones going straight to the file scope (which is most of them) to use `getFileScope()` instead.  This will allow us to more easily change how the file scope is accessed if e.g. we move it into the decl directly or we move it to only the root namespace or something.  It also makes the codebase more searchable in general.